include/recognition/CographAlg.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <optional>
+#include <vector>
+#include <networkit/base/Algorithm.hpp>
+#include <networkit/components/ConnectedComponents.hpp>
+#include <networkit/graph/Graph.hpp>
+
+#include "structures/Cotree.hpp"
+#include "Cograph/FactorizingPermutation.hpp"
+#include "Cograph/Part.hpp"
```

Comment:
I guess `cograph` should be directory name + CographRecognition.hpp the filename, so it'd conform to the previous ones.